tests/cases/draw_rect_multiple_pipeline.amber

```diff
@@ -39,6 +39,6 @@ RUN my_pipeline DRAW_RECT POS 0 0 SIZE 250 250
 RUN pipeline2 DRAW_RECT POS 250 250 SIZE 250 250
 RUN pipeline3 DRAW_RECT POS 0 250 SIZE 250 250
 
-EXPECT framebuffer IDX 250 250 SIZE 250 250 EQ_RGBA 0 127 0 204
```

Comment:
Instead of modifying this example, let's duplicate it and create `draw_rect_multiple_pipeline_with_tolerance.amber`.

Reply:
That would result in a bunch of tests that fail on at least two different vendor's driver. Why would we keep tests that are known to be prone to fail because of precision?

Reply:
I didn't realize it failed on various drivers, so keeping it with the tolerance makes sense.